collectors/proc.plugin/multi_metadata.yaml

```diff
@@ -3642,20 +3685,22 @@ modules:
       plugin_name: proc.plugin
       module_name: /proc/net/rpc/nfsd
       monitored_instance:
-        name: proc /proc/net/rpc/nfsd
+        name: NFS Server
         link: ''
-        categories: []
-        icon_filename: ''
+        categories:
+         - data-collection.linux-systems.filesystem-metrics.nfs
+        icon_filename: 'syslog.png'
```

Comment:
Just like with the NFS client, this should probably be a filesystem or network related icon instead.